### PR TITLE
Bump enet library version

### PR DESCRIPTION
**Pull request by tpimh**
Also changed the link from github to project website and installing with `make install` now.

### CODE REVIEW

**Review by sharkwouter — CHANGES_REQUESTED** [review]
With this change `libenet.a` ends up in `$PSPDEV/psp/lib/static` instead of `$PSPDEV/psp/lib`. This should be fixed before merging.

**Comment by fjtrujy** [comment]
What's the motivation to change from GitHub to his official website? 
In this case, since the official repo is still on GitHub (check issues and PRs), I think it is safer and actually better to keep using GitHub (we can't assure the website to be available as much as Github)

File: enet/PSPBUILD [hunk 1]
```diff
@@ -1,15 +1,15 @@
 pkgname=enet
-pkgver=1.3.17
-pkgrel=2
+pkgver=1.3.18
+pkgrel=1
```
Comment (bucanero) [inline enet/PSPBUILD:3]:
```suggestion
pkgrel=3
```

 bump `pkgrel` up

Comment (tpimh) [reply enet/PSPBUILD:3]: I think we need to reset `pkgrel` back to 1 when updating `pkgver`. Correct me if I'm wrong.

Comment (sharkwouter) [reply enet/PSPBUILD:3]: Yeah, pkgrel at 1 is correct

**Comment by tpimh** [comment]
> What's the motivation to change from GitHub to his official website? In this case, since the official repo is still on GitHub (check issues and PRs), I think it is safer and actually better to keep using GitHub (we can't assure the website to be available as much as Github)

Yes, I just started thinking about it too, because HTTPS... Should probably change it back.

**Comment by sharkwouter** [comment]
I guess changing it back to use github is a bit better. Otherwise this is good now.

**Comment by sharkwouter** [comment]
Thanks! Looks good now.

**Pull request merged**